rfcs/062-knowledge-graph/README.md

```diff
@@ -0,0 +1,234 @@
+    "stories": stories,
+    "story_ids": story_ids,
+    "variants": variants,
+}
+```
+If a source node is present, the following is added depending on the source:
+```
+# MeSH
+{
+    "mesh_description": mesh_source.description,
+    "mesh_id": mesh_source.source_id,
+    "mesh_preferred_label": mesh_source.preferred_label,
+    "preferred_label": mesh_source.preferred_label,
+}
+# Library of Congress
+{
+    "lc_names_id": lc_names_source.source_id,
+    "lc_names_preferred_label": lc_names_source.preferred_label,
+    "preferred_label": lc_names_source.preferred_label,
+    "preferred_label_source": "lc-names",
+}
+# or
+{
+    "lc_subjects_id": lc_subjects_source.source_id,
+    "lc_subjects_preferred_label": lc_subjects_source.preferred_label,
+    "preferred_label": lc_subjects_source.preferred_label,
+    "preferred_label_source": "lc-subjects",
+}
+# Wikidata
+{
+    "wikidata_description": wikidata_source.description,
+    "wikidata_id": wikidata_source.source_id,
+    "wikidata_preferred_label": wikidata_source.preferred_label,
+    "preferred_label": wikidata_source.preferred_label,
+    "preferred_label_source": "wikidata",
+}
+```
+
+## Next steps: Wellcome Collection Graph (v2)
+
+Here is a brief overview of things to consider when developing v2 of a graph for Wellcome Collection:
+
+**Data sources**
+
+Ensure that local data sources like spreadsheets are excluded. Verify that URLs of external data sources are current and functional, and consider schema verification for input data. Reassess external data acquisition methods, comparing API queries and full data downloads. Explore potential additional data sources and confirm that all relevant information is being captured. Consider update schedules of the different data sources and how these feed into graph updates. 
+
+**Graph model**
+
+Re-evaluate the graph model and make sure it represents internal and external data effectively. Ensure the graph model is efficient and supports downstream tasks like indexing, machine learning analyses, and visualisation. Verify that edge directionality aligns with the edge labels, such as reversing HAS_CONCEPT edges to reflect semantic directionality. Consider whether undirected edges should exist between source concept nodes.
+
+**Pipeline**
+
+Apply learnings from the Wellcome Academic Graph's pipeline design where applicable. For example, consider using the Neo4j driver with Cypher queries to load data to Neo4j and removing the neomodel dependency for better performance, leveraging existing expertise from the machine learning team. Refactoring changes to increase pipeline performance may also include preprocesssing all source data into nodes and edges before bulk loading them to Neo4j, instead of enriching and loading each work iteratively. Finally, ensure the code meets machine learning team standards by improving Python import readability, adding correctly formatted docstrings, enabling auto-formatting, and adding unit tests.
```

Comment:
```suggestion
Apply learnings from the Wellcome Academic Graph's pipeline design where applicable. For example, consider using the Neo4j driver with Cypher queries to load data to Neo4j and removing the neomodel dependency for better performance, leveraging existing expertise from the machine learning team. Refactoring changes to increase pipeline performance may also include preprocessing all source data into nodes and edges before bulk loading them to Neo4j, instead of enriching and loading each work iteratively. Finally, ensure the code meets machine learning team standards by improving Python import readability, adding correctly formatted docstrings, enabling auto-formatting, and adding unit tests.
```